lib/round.rb

```diff
@@ -0,0 +1,45 @@
+class Round
+  attr_reader :deck, :turns
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+  end
+
+  def current_card
+    deck.cards[turns.count]
```

Comment:
Be careful when referencing instance variables that you use the `@` sign consistently.